packages/examples/cvat/exchange-oracle/src/handlers/job_creation.py

```diff
@@ -1234,16 +1210,14 @@ def _draw_roi_point(
             (255, 255, 255),
             cv2.FILLED,
         )
-        roi_pixels = cv2.circle(
+        return cv2.circle(
```

Comment:
I think in some cases this optimization makes it a little bit harder to understand what's being returned.

Reply:
I'd argue that is should be signified by the function name and return annotation.

In this example, I don't see how removing intermediate variable is confusing.

Otherwise, to me it would make more sense to rename `_draw_roi_point` -> `_get_roi_pixels`. Though current name is clear enough, IMO.